ls1intum/Artemis
# [Enhancement][Exam] Minor Client Improvements

### Checklist

### Motivation and Context
To keep the create buttons consistently and to provide better user feedback. Also, reduced the whitespace for the code editor

### Description
Refactored the submit button text display to only be handled in the HTML file using the translations
Changed the error message on noExam to reflect the user not being registered and the exam title.
refactored the code editor title and clone button placement


### Steps for Testing

1. Log in to Artemis
2. Try out create/edit/import text exercises. The submit button should be displayed as create, edit and import respectively
--------------------------------------------
1. Log in to Artemis
2. Create an exam and set the start and end time and don't sign up as a student
3. navigate to the exam. You should now see the message that you are not registered
---------------------------------------------
1. Log in to Artemis
2. Create an exam with a code editor and clone button exercise and start it
3. navigate to the exercise, the whitespace should look decent now and the button should be beside the title.
---------------------------------------------
### Screenshot
![Capture](https://user-images.githubusercontent.com/15110960/86165390-80ecd980-bb13-11ea-9ee4-431cc8a32b7d.PNG)
